### PR TITLE
feat(send): show warning for unknown addresses

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Mainly the UI, the condition for address will be updated in a followup once address lookup is ready (after #4586)

### Test plan

Unit tests, manually by searching for different types of recipients

https://github.com/valora-inc/wallet/assets/5062591/d8ac828b-bdeb-43cb-82a7-814c61c19c27

### Related issues

- Part of ACT-979

### Backwards compatibility

Yes

### CODE REVIEW

File: src/send/SendSelectRecipient.tsx [hunk 1]
```diff
@@ -408,8 +426,12 @@ const styles = StyleSheet.create({
     padding: Spacing.Thick24,
     textAlign: 'center',
   },
+  unknownAddressInfo: {
+    padding: Spacing.Thick24,
+    paddingBottom: Spacing.Smallest8,
+  },
   sendOrInviteButton: {
```
[the remark this answers is not in the document]
Comment (satish-ravi) [reply src/send/SendSelectRecipient.tsx:437]: updated, but I think we use padding for most elements in this file

File: src/components/InLineNotification.tsx [hunk 2]
```diff
@@ -81,7 +81,6 @@ const styles = StyleSheet.create({
   container: {
     padding: Spacing.Regular16,
     borderRadius: Spacing.Regular16,
-    width: '100%',
```
Comment (satish-ravi) [inline src/components/InLineNotification.tsx:84]: This was causing the notification to overflow if margins are used in the style prop. This was actually not required, I've tested every usage of this including
- SendEnterAmount
- SwapScreen
- TokenImport
- ActionRequest
- SessionRequest

File: src/send/SendSelectRecipient.tsx [hunk 3]
```diff
@@ -201,6 +202,14 @@ function SendSelectRecipient({ route }: Props) {
   const { recipientVerificationStatus, recipient, setSelectedRecipient, unsetSelectedRecipient } =
     useFetchRecipientVerificationStatus()
 
+  // TODO(satish/joe): update condition to not show this if the address
```
Comment (MuckT) [inline src/send/SendSelectRecipient.tsx:205]: Does this have an associated ticket?

**Review by MuckT — APPROVED** [review]
Approved with a question about the follow up ticket for showUnknownAddressInfo.

File: src/components/InLineNotification.tsx (hunk 2 above)
Comment (MuckT) [reply src/components/InLineNotification.tsx:84]: Double checked on ActionRequest and LGTM.

File: src/send/SendSelectRecipient.tsx (hunk 3 above)
Comment (satish-ravi) [reply src/send/SendSelectRecipient.tsx:205]: ACT-979 (same as this ticket), will do a fast follow once #4586 is merged

**Pull request merged**